server/container_create_linux.go

```diff
@@ -969,6 +969,9 @@ func addOCIBindMounts(ctx context.Context, ctr ctrfactory.Container, mountLabel,
 		if m.HostPath == "/" && dest == "/" {
 			log.Warnf(ctx, "Configuration specifies mounting host root to the container root.  This is dangerous (especially with privileged containers) and should be avoided.")
 		}
+		if m.HostPath == "/var/lib/containers" {
+			m.Propagation = types.MountPropagation_PROPAGATION_HOST_TO_CONTAINER
```

Comment:
can we log an info to let the user know what's happening? 

I also wonder if this should be configurable to start... wdyt @cri-o/cri-o-maintainers

Reply:
Can we infer `/var/lib/containers` from an existing configuration?

Reply:
I am inclined to agree with @saschagrunert here. I would not say the priority here is high enough to not have a complete solution.
using https://pkg.go.dev/path/filepath#Rel may be a possibility to calculate whether a path contains the storage root

Reply:
alternatively, maybe
`strings.Contains(storage.Root, m.HostPath)` may give us what we want
https://go.dev/play/p/OU5jiFvElL3